templates/prescription/general_edit.html

```diff
@@ -160,8 +160,7 @@
 <tr>
   <td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xl t='Starting Date'|escape:'html'}</td>
   <td COLSPAN="2" ALIGN="LEFT" VALIGN="MIDDLE" >
-    {html_select_date class="input-sm" start_year="-10" end_year="+5" time=$prescription->start_date prefix="start_date_"}
-    <input type="hidden" name="start_date" value="{$prescription->start_date|escape:'html'}">
+      <input   type="text" size="10" class="datepicker" name="start_date" id="start_date" value="{$prescription->start_date|oeFormatShortDate|escape:'html'}" />
```

Comment:
Does oeFormatShortDate work here???

Reply:
You'll need to create the smarty function here:
(check out the others for examples)
https://github.com/openemr/openemr/tree/master/library/smarty/plugins

Reply:
Yes, It's working..

Reply:
That hurts my brain (since there is no smarty plugin, it shouldn't work; at least I don't think it should). What happens if you remove it? Does anything change in testing?